Huginn/chains.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "../chains.schema.json",
+    "name": "Huginn",
+    "chains": [
+      {
+        "name": "Passage",
```

Comment:
These probably need to be lowercase - they should match the directory names in chain registry. It might work like this but I'm not 100% sure so it's better to change them